astroid/rebuilder.py

```diff
@@ -61,6 +61,11 @@ def __init__(
         self._manager = manager
         self._data = data.split("\n") if data else None
         self._global_names: list[dict[str, list[nodes.Global]]] = []
+        # In _nonlocal_names,
```

Comment:
This comment is formatted a bit strange. Does it fit on two lines?

Reply:
I've modified it to

```python
# In _nonlocal_names saves the FunctionDef where the variable is created,
# rather than Nonlocal, since we don't really need the Nonlocal statement.
```